examples/h3/README.md

```
@@ -0,0 +1,41 @@
+    pnpm install
+    ~~~~
+
+4.  Start the server:
+
+    ~~~~ sh
+    deno serve -A index.ts
+    ~~~~
+
+5.  Look up the actor URL:
+
+    ~~~~
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)